bci_tester/data.py

```diff
@@ -415,8 +415,8 @@ def create_BCI(
             extra_marks=[pytest.mark.__getattr__(f"golang_{stability}")],
         )
         for golang_version, stability in (
-            ("1.20", "oldstable"),
-            ("1.21", "stable"),
+            ("1.21", "oldstable"),
```

Comment:
I wonder if we shouldn't finally go with oldstable and stable to not have to do that every 6 months